lib/gcloud/dns/zone.rb

```diff
@@ -360,6 +363,7 @@ def records options = {}
           fail ApiError.from_response(resp)
         end
       end
+      alias_method :find_record, :records
```

Comment:
@quartzmo I've addressed this in a [separate PR](https://github.com/GoogleCloudPlatform/gcloud-ruby/pull/335).